__DEFINES/_macros.dm

```diff
@@ -196,6 +196,8 @@
 
 #define istable(A) (istype(A, /obj/structure/table))
 
+#define isholder(A) (istype(A, /obj/structure/table) || istype(A, /obj/structure/rack) || istype(A, /obj/structure/closet) || istype(A, /obj/item/weapon/storage))
```

Comment:
not your fault, you don't need to fix it, but wow this macro name sucks

Reply:
I really struggled to find a good name for "thing you can put other thing into/onto". I'm up for suggestions.

Reply:
isshelf perhaps; this is defined as a "jutting flat area or piece" and I think "shelf" communicates "place to put things on"

MOST importantly it does not confuse with holder datums

Reply:
container possibly, but a tabletop isn't a container